Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1745,8 +1765,22 @@ def lhegensim2018ml(fragment,howMuch):
 
 steps['DIGIPRMXUP15_PU50']=merge([digiPremixUp2015Defaults50ns])
 steps['DIGIPRMXUP17_PU25']=merge([digiPremixUp2017Defaults25ns])
+steps['DIGIPRMXUP17_PU25_RD']=merge([digiPremixUp2017Defaults25ns, { '--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosMCRcd'),  tag = cms.string('EcalLaserAPDPNRatios_UL_2017_mc'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) )  \\n process.ecal_sim_parameter_map.timeDependent=cms.bool(True)\"" } ])
 steps['DIGIPRMXLOCALUP17_PU25']=merge([digiPremixLocalPileupUp2017Defaults25ns])
 steps['DIGIPRMXUP18_PU25']=merge([digiPremixUp2018Defaults25ns])
+steps['DIGIPRMXUP18_PU25_RD']=merge([digiPremixUp2018Defaults25ns, { '--customise_commands':"\"process.EcalLaserCorrectionServiceMC = cms.ESProducer('EcalLaserCorrectionServiceMC') \\n process.GlobalTag.toGet = cms.VPSet( cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosMCRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ), cms.PSet(     record = cms.string('EcalLaserAPDPNRatiosRcd'),  tag = cms.string('EcalLaserAPDPNRatios_Run_Dep_MC'),  connect = cms.string('frontier://FrontierProd/CMS_CONDITIONS') ) )  \\n process.ecal_sim_parameter_map.timeDependent=cms.bool(True) \\n process.source.setRunNumberForEachLumi = cms.untracked.vuint32(1,316083)\"" } ])
+#
+#* EcalPedestals_Run_Dep_MC                 (9IOVs)   run numbers: 1, 316082, 316720, 317527, ...
+#* EcalLaserAPDPNRatios_Run_Dep_MC          (9IOVs)   
+#* EcalLaserAlphas_Run_Dep_MC               (1IOV)    
+#* EcalADCToGeVConstant_UL_2018_v1_mc       (1IOV)    
+#* EcalChannelStatus_UL_2018_v2_mc          (1IOV)    
+#* EcalIntercalibConstants_forTimeDep_mc    (1IOV)
+#* EcalIntercalibConstantsMC_forTimeDep_mc  (1IOV)
+#* EcalPFRecHitThresholds_UL_2018_2e3sig_v2 (1IOV)
+#
+#* EcalLaserAPDPNRatios_Run_Dep_MC_first_IOV   (1 IOV)   ---> used in SIM?
+#
```

Comment:
what are L1772-L1783??

Reply:
ok, I will do the cleaning together with the fix proposed in the configuration